apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/Internal/Logging/Logger.cs

```diff
@@ -106,6 +106,23 @@ public void ResetPerformanceData()
         /// <param name="entries">The entries to log. This is never null, and never empty.</param>
         protected abstract void LogPerformanceEntries(IEnumerable<string> entries);
 
+        /// <summary>
+        /// This method is called when a transaction that requested CommitStats is committed.
+        /// The default implementation logs the commit stats at log level Debug. Derived classes
+        /// can override this method to log the statistics at a different level, or to a different
+        /// destination.
+        /// </summary>
+        /// <param name="request">The commit request that requested commit statistics</param>
+        /// <param name="response">The response with commit statistics</param>
+        public virtual void LogCommitStats(CommitRequest request, CommitResponse response)
+        {
+            if (response.CommitStats != null)
+            {
+                Debug(() => $"Transaction {request.TransactionId?.ToBase64() ?? ""} mutation count: {response.CommitStats.MutationCount}");
```

Comment:
I think I'd use Info rather than Debug here, given that this is already opt-in information.

Reply:
Done.